### ISSUE TITLE
Accessibility: Submit button do not have a sufficient contrast ratio.

### ISSUE THREAD

**Issue by Pareesh** [hlxsites/mammotome#510]
The submit button does not have proper contrast ratio, hence making it difficult or impossible for many users to read.

**Comment by Pareesh** [comment]
Fixed with commit https://github.com/hlxsites/mammotome/pull/508
